client/src/components/RelatedItemsAndOutfit/UserOutfits/AddOutFitCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AddOutfitCard = ({ addToCurrOutfits }) => {
+  const makeLintHappy = addToCurrOutfits;
```

Comment:
I didn't have to declare a new const with a function passed as a prop, this linter behavior is interesting

Reply:
Try removing line 5 and change line 9 onClick to onClick={() => {addToCurrOutfits(yourParams)}}

Reply:
could also be b/c proptypes is expecting syntax like "PropTypes.func.isRequired" vs. instanceOf(Function)?

Reply:
Oh on i just have this for testing purposes, I don't need the line hahaha